hexa/plugins/connector_accessmod/graphql/schema.graphql

```diff
@@ -191,10 +195,14 @@ type AccessmodAccessibilityAnalysis implements AccessmodAnalysis {
     barrier: AccessmodFileset
     movingSpeeds: AccessmodFileset
     healthFacilities: AccessmodFileset
-    anisotropic: Boolean
     invertDirection: Boolean
     maxTravelTime: Int
-
+    maxSlope: Float
+    priorityRoads: Boolean
+    priorityLandCover: [String]
+    waterAllTouched: Boolean
+    algorithm: AccessmodAccessibilityAnalysisAlgorithm
+    knightMove: Boolean
     travelTimes: AccessmodFileset
     frictionSurface: AccessmodFileset
```

Comment:
Not a priority but the pipeline is also creating a 3rd fileset `CatchmentAreas` (raster)